src/vid.rs

```diff
@@ -70,9 +72,16 @@ pub type VidCommon = <VidSchemeType as VidScheme>::Common;
 /// VID share type
 pub type VidShare = <VidSchemeType as VidScheme>::Share;
 
+#[cfg(not(feature = "gpu-vid"))]
+/// Internal Jellyfish VID scheme
+type VidInternal = Advz<E, H>;
+#[cfg(feature = "gpu-vid")]
+/// Internal Jellyfish VID scheme
+type VidInternal = AdvzGPU<'static, E, H>;
+
```

Comment:
personal preference:

```suggestion
#[cfg(not(feature = "gpu-vid"))]
/// Internal Jellyfish VID scheme
type Advz = advz::Advz<E, H>;
#[cfg(feature = "gpu-vid")]
/// Internal Jellyfish VID scheme
type Advz = advz::AdvzGPU<'static, E, H>;
```

Because the `VidInternal` sounds like something Jellyfish happens to expose as low level details.